rollup.config.js

```diff
@@ -15,6 +16,7 @@ export default {
     }
   ],
   plugins: [
+    json(),
```

Comment:
Do we need this JSON plugin installed if we're using WebPack for the UMD build?

Reply:
Yes, because the rollup does not import json. As I changed from "require" to "import" we need this

Reply:
Not sure I understand but as long as you do

Reply:
because of this change:
<img width="943" alt="Captura de Tela 2022-08-24 às 11 29 22" src="https://user-images.githubusercontent.com/1014611/186444930-7c2de1a6-7b15-466d-97b5-2e3ffad7b603.png">

Rollup does not include "require" in the build. To include in the build, we must import as "import" and use this plugin because the "import" is a json.

Reply:
Ah, thank you for explaining.